johnwayodi/sm-api
# #161498848 update api documentation to, add json validation

#### What does this PR do?
Updates the documentation of the API and adds json validation to post requests
#### Description of Task to be completed?
Add json validation on sale record creation 
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, sm-api by default.
3. Create a virtual environment using virtualenv and pip install -r requirements.txt
4. `git checkout ch-update-api-docs-161498848`
5. Run application and head over to `/apidocs` to view dicumentation.
#### What are the relevant pivotal tracker stories?
[#161498848](https://www.pivotaltracker.com/story/show/161498848)
#### Questions:
Any feedback that I can improve, leave a suggestion below.
